html/semantics/embedded-content/the-img-element/sizes/sizes-iframed.sub.html

```diff
@@ -52,8 +52,14 @@
 <img srcset='/images/green-1x1.png?e34 50w, /images/green-16x16.png?e34 51w' sizes='\[,1px'>
 <img srcset='/images/green-1x1.png?e35 50w, /images/green-16x16.png?e35 51w' sizes='1\p\x'>
 <img srcset='/images/green-1x1.png?e36 50w, /images/green-16x16.png?e36 51w' sizes='calc(1px)'>
+<img srcset='/images/green-1x1.png?e36 50w, /images/green-16x16.png?e36 51w' sizes='min(1px, 100px)'>
```

Comment:
The number in the query should be unique so that we avoid having the browser prefer to load a candidate based on it already being in cache. You can use 36a or so.